### DISCUSSION TITLE
illa-cli本地部署的，localhost:1000能访问首页，但登录的http接口发不通，貌似是9999端口的服务没起起来？

### DISCUSSION THREAD

**Discussion by OldChan** [illacloud/illa-builder#848] [Q&A] [answered]
1、使用illa-cli本地部署的，illa deploy --self --port=10000 --server-addr=localhost 执行成功（ILLA Builder started, please visit http://localhost:10000）
2、浏览器访问自动跳转到登录页，登录接口报POST http://localhost:9999/api/v1/auth/signin net::ERR_EMPTY_RESPONSE

请问有遇到过这个问题吗？

**Comment by karminski** [comment, marked as answer]
@OldChan 你好, 可以使用 docker logs 输出一下日志看一下吗?

**Reply by OldChan** [reply]
> @OldChan 你好, 可以使用 docker logs 输出一下日志看一下吗?

您好，感谢回复。docker logs打印出下面一个错：
2023/02/08 00:56:53 /opt/illa/builder-backend/pkg/db/connection.go:61
[error] failed to initialize database, got error failed to connect to `host=localhost user=illa database=illa`: dial error (dial tcp [::1]:5432: connect: cannot assign requested address)
{"level":"error","ts":1675817813.3594499,"caller":"zap@v1.24.0/sugar.go:222","msg":"Ignored key without a value.","ignored":1,"stacktrace":"go.uber.org/zap.(*SugaredLogger).Errorw\n\t/go/pkg/mod/go.uber.org/zap@v1.24.0/sugar.go:222\ngithub.com/illacloud/builder-backend/pkg/db.NewDbConnection\n\t/opt/illa/builder-backend/pkg/db/connection.go:56\nmain.Initialize\n\t/opt/illa/builder-backend/cmd/http-server/wire_gen.go:38\nmain.main\n\t/opt/illa/builder-backend/cmd/http-server/main.go:20\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:250"}
{"level":"error","ts":1675817813.359402,"caller":"db/connection.go:56","msg":"Failed to connect to database, %d","stacktrace":"github.com/illacloud/builder-backend/pkg/db.NewDbConnection\n\t/opt/illa/builder-backend/pkg/db/connection.go:56\nmain.Initialize\n\t/opt/illa/builder-backend/cmd/http-server/wire_gen.go:38\nmain.main\n\t/opt/illa/builder-backend/cmd/http-server/main.go:20\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:250"}
panic: failed to connect to `host=localhost user=illa database=illa`: dial error (dial tcp [::1]:5432: connect: cannot assign requested address)

goroutine 1 [running]:
github.com/illacloud/builder-backend/pkg/db.NewDbConnection(0xc00013d220, 0xc0000a87f8)
        /opt/illa/builder-backend/pkg/db/connection.go:68 +0x908
main.Initialize()
        /opt/illa/builder-backend/cmd/http-server/wire_gen.go:38 +0xa5
main.main()
        /opt/illa/builder-backend/cmd/http-server/main.go:20 +0x19

**Reply by karminski** [reply]
hi, 有在5432端口上启动其他服务吗? 看样子是 5432端口被占用或者没有权限. 没有权限可以尝试使用root用户执行部署.
